deployments/liqo/values.yaml

```diff
@@ -9,8 +9,8 @@ pullPolicy: "IfNotPresent"
 apiServer:
   # -- The address that must be used to contact your API server, it needs to be reachable from the clusters that you will peer with (defaults to your master IP)
   address: ""
-  # -- The port that must be used to contact your API server
-  port: "6443"
+  # -- The port that must be used to contact your API server (if not yet specified in the address field)
+  port: ""
```

Comment:
From the user perspective, I would personally keep only the address field, defaulting to the master IP address and port 6443 in the other cases. Having both parameters makes it unclear what to put in the port one and might lead to configuration errors. 

That would also simplify the logic below. If the variable is set you just use that (possibly prepending https://), otherwise you gather the master IP and construct the value.